docs/docs/language_reference/schema/index.md

```diff
@@ -116,19 +116,272 @@ type Post implements Webpage {
 
 ## Vertex types and properties
 
+A [vertex][vertex] is akin to a table in SQL, given our Flutter example where
+we have users and they may post things, or follow other users we can see we have
+two clear vertices. A User vertex and a Post vertex.
```

Comment:
The sentence structure is a bit unclear, possibly because of some editing that's still in progress?